### PR TITLE
BUG: df.pivot_table: margins_name ignored when aggfunc is a list

### LINKED ISSUE #13354: BUG: df.pivot_table: margins_name is ignored when there aggfunc is list
#### Code Sample, a copy-pastable example if possible
```
costs = pd.DataFrame({'item':['bacon','cheese','bacon','cheese'],
                      'cost':[2.5,4.5,3.2,3.3],
                     'store':['SuperS','DollarM','SuperS','DollarM'],
                     'day':['M','M','T','T']})
costs.pivot_table(index="item",columns="day",margins=True, margins_name="Weekly",aggfunc=[np.mean,max])
```
Will ignore the margins_name

#### Expected Output
```
      mean               max                                     
       cost              cost              store                  
          M     T    Weekly    M    T  Weekly        M        T      Weekly
item                                                              
bacon   2.5  3.20  2.850  2.5  3.2  3.2   SuperS   SuperS   SuperS
cheese  4.5  3.30  3.900  4.5  3.3  4.5  DollarM  DollarM  DollarM
All     3.5  3.25  3.375  4.5  3.3  4.5   SuperS   SuperS   SuperS
```
#### output of ``pd.show_versions()``
```
INSTALLED VERSIONS
------------------
commit: None
python: 3.4.3.final.0
python-bits: 64
OS: Linux
OS-release: 3.13.0-43-generic
machine: x86_64
processor: x86_64
byteorder: little
LC_ALL: None
LANG: en_US.UTF-8

pandas: 0.18.1
nose: None
pip: 1.5.6
setuptools: 20.2.2
Cython: None
numpy: 1.11.0
scipy: 0.17.1
statsmodels: None
xarray: None
IPython: 4.1.2
sphinx: None
patsy: None
dateutil: 2.5.3
pytz: 2016.3
blosc: None
bottleneck: None
tables: None
numexpr: None
matplotlib: 1.5.1
openpyxl: None
xlrd: None
xlwt: None
xlsxwriter: None
lxml: None
bs4: None
html5lib: 0.9999999
httplib2: 0.9.2
apiclient: 1.5.0
sqlalchemy: None
pymysql: None
psycopg2: None
jinja2: 2.8
boto: None
pandas_datareader: None
```

### PR DESCRIPTION
**Pull request by cmazzullo**
- [ ] closes #13354 
- [ ] tests added / passed
- [ ] passes `git diff upstream/master | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: doc/source/whatsnew/v0.18.1.txt [hunk 1]
```diff
@@ -682,3 +682,4 @@ Bug Fixes
 - Bug in ``pd.to_numeric()`` with ``Index`` returns ``np.ndarray``, rather than ``Index`` (:issue:`12777`)
 - Bug in ``pd.to_numeric()`` with datetime-like may raise ``TypeError`` (:issue:`12777`)
 - Bug in ``pd.to_numeric()`` with scalar raises ``ValueError`` (:issue:`12777`)
+- Bug in ``pd.pivot_table()`` where margins_name is ignored when aggfunc is a list (:issue:`13354`)
```
Comment (TomAugspurger) [inline doc/source/whatsnew/v0.18.1.txt]: Could you move this to `whatsnew/0.18.2.txt` :)

**Comment by TomAugspurger** [comment]
Could you also add a test ensuring that the margins have the correct names? Thanks.

**Comment by cmazzullo** [comment]
Sure thing!

Chris Mazzullo

File: pandas/tools/tests/test_pivot.py [hunk 2]
```diff
@@ -779,6 +779,13 @@ def test_pivot_table_with_iterator_values(self):
         )
         tm.assert_frame_equal(pivot_values_gen, pivot_values_list)
 
+    def test_pivot_table_margins_name_with_aggfunc_list(self):
+        # GH 13354
+        margins_name = 'Weekly'
+        table = self.data.pivot_table(index='A', columns='B', margins=True,
+                              margins_name=margins_name, aggfunc=[np.mean, max])
+        self.assertEqual(table[:][:].index[2], margins_name)
```
Comment (sinhrks) [inline pandas/tools/tests/test_pivot.py]: pla compare whole `DataFre` preparing expected output.

File: doc/source/whatsnew/v0.18.2.txt [hunk 3]
```diff
@@ -383,3 +383,4 @@ Bug Fixes
 
 
 - Bug in ``Categorical.remove_unused_categories()`` changes ``.codes`` dtype to platform int (:issue:`13261`)
+- Bug in ``pd.pivot_table()`` where margins_name is ignored when aggfunc is a list (:issue:`13354`)
```
Comment (jreback) [inline doc/source/whatsnew/v0.18.2.txt]: margine_name in double back-ticks

File: pandas/tools/tests/test_pivot.py [hunk 4]
```diff
@@ -779,6 +779,28 @@ def test_pivot_table_with_iterator_values(self):
         )
         tm.assert_frame_equal(pivot_values_gen, pivot_values_list)
 
+    def test_pivot_table_margins_name_with_aggfunc_list(self):
+        # GH 13354
+        margins_name = 'Weekly'
```
Comment (jreback) [inline pandas/tools/tests/test_pivot.py]: add a comment about what this is testing

**Comment by jreback** [comment]
thanks!

**Pull request closed without merging**